test/ScriptCs.Tests.Acceptance/LooseScriptExecution.cs

```diff
@@ -0,0 +1,96 @@
+﻿namespace ScriptCs.Tests.Acceptance
+{
+    using System;
+    using System.Reflection;
+    using ScriptCs.Tests.Acceptance.Support;
+    using Should;
+    using Xbehave;
+    using Xunit;
+
+    public static class LooseScriptExecution
+    {
+        [Scenario]
+        [Example(true)]
+        [Example(false)]
+        public static void HelloWorld(bool debug, ScenarioDirectory directory, string output, string[] args, string script)
+        {
+            var scenario = MethodBase.GetCurrentMethod().GetFullName();
+
+            "Given a hello world script"
+                .f(() =>
+                {
+                    directory = ScenarioDirectory.Create(scenario);
+                    if (Environment.OSVersion.Platform == PlatformID.Win32NT)
+                    {
+                        script = @"Console.WriteLine(""""""Hello World!"""""");";
```

Comment:
Because it does. This has been thoroughly tested and it is apparently by design.